code/game/objects/items/RPD.dm

```diff
@@ -479,12 +479,28 @@ GLOBAL_LIST_INIT(transit_tube_recipes, list(
 					if (target_differences & S.connections)
 						to_chat(user, span_warning("\The [src]'s screen flashes a warning: Can't configure a pipe in a currently connected direction."))
 						return
-					var/new_dir = (S.GetInitDirections() & ~target_differences) | target_differences
+					var/old_dir = S.GetInitDirections()
+					var/new_dir = (old_dir & ~target_differences) | target_differences
```

Comment:
Which version of this line is correct? #61136

Reply:
Neither. This doesn't change or fix any functionality at these lines, it just splits out `old_dir` to be reused further down. This PR can be applied first, the other PR needs fixing.